test/double_mustcall/index.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "../../Annotations/Annotations.h"
+
+struct M {
+    char* x MustCall("free, free1");
+    char* y MustCall("free");
+};
+
+void free1(void* p Calls("free1")) {
```

Comment:
Not really related to this PR, but I think I just realized that I don't understand the semantics of your `Calls` annotation. I would have expected the annotation here to be `Calls("free")` instead of `Calls("free1")`, since this method calls `free()` on `p`; this method _is_ `free1`, so it doesn't make sense to write that it calls itself on its own argument (which seems to me to be a tautology). We should discuss this (maybe in our next meeting or over email).

Reply:
This is a great point! Logic fix coming soon in a new pr. I agree, it shouldn't make sense that the code works this way. Its purpose here is to try to say that `p` calls `free1`, but the code should (and is definitely able to) pick that up automatically whenever we actually call `free1`. `Call("free")` would be the appropriate annotation on `p`.